### PR TITLE
Minor README updates

### PR DESCRIPTION
**Pull request by msaroufim**
Minor things that were missed from https://github.com/pytorch/ao/pull/392

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ All with no intrusive code changes and minimal accuracy degradation.
 
 #### Without intrusive code changes
 
-Quantizing your models is a 1 liner that should work on any model with `nn.Linear` including your favorite HuggingFace model. You can find a more comprehensive usage instructions [here](torchao/quantization/) and a hugginface inference example [here](scripts/hf_eval.py)
+Quantizing your models is a 1 liner that should work on any model with an `nn.Linear` including your favorite HuggingFace model. You can find a more comprehensive usage instructions [here](torchao/quantization/) and a HugginFace inference example [here](scripts/hf_eval.py)
```
Comment (supriyar) [inline README.md:18]: HuggingFace

File: README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 ## Introduction
 
-torchao is a library to create and integrate high-performance custom data types, layouts and kernels into their PyTorch workflows with up to **2x speedups** with **65%** less VRAM for [inference](#inference) and support for [training](#training)
+torchao is a library to create and integrate high-performance custom data types, layouts and kernels into your PyTorch workflows with up to **2x speedups** with **65% less VRAM** for [inference](#inference) and support for [training](#training)
```
Comment (supriyar) [inline README.md:8]: should we also have an index like thing at the top like tune does? https://github.com/pytorch/torchtune?tab=readme-ov-file#torchtune

File: README.md [hunk 3]
```diff
@@ -93,6 +91,13 @@ Nightly Release
...
 ```
 
+From source 
+```Shell
+git clone https://github.com/pytorch/ao
+cd ao
+python setup.py install
+```
+
 ## Community Contributions
```
Comment (supriyar) [inline README.md:102]: typo in this section that I missed before uin2 -> uint2

File: README.md (hunk 2 above)
Comment (msaroufim) [reply README.md:9]: Done

File: README.md [hunk 4]
```diff
@@ -73,10 +71,10 @@ swap_linear_with_semi_sparse_linear(model, {"seq.0": SemiSparseLinear})
...
 1. Write the dtype, layout or bit packing logic in pure PyTorch and code-generate efficient kernels with torch.compile. You can inspect those kernels with `TORCH_LOGS="output_code" python your_code.py` and check if a single kernel is being generated and if any unnecessary buffers are being created
 2. However once you get a kernel, how do you know how good it is? The best way is to benchmark the code-generated code with the best kernel on the market. But packaging custom CPP/CUDA kernels that work on multiple devices is tedious but we've abstracted all the tedium from you with our [custom ops support](./torchao/csrc/) so if you love writing kernels but hate packaging, we'd love to accept contributions for your custom ops. One key benefit is a kernel written as a custom op will just work with no graph breaks with `torch.compile()`. Compilers are great at optimizations like fusions and overhead reduction but it's challenging for compilers to rewrite the math of an algorithm such that it's faster but also numerically stable so we are betting on both compilers and custom ops
```
Comment (supriyar) [inline README.md:77]: code-generated code reads a bit funny. Maybe you mean compiler generated code?

**Review by supriyar — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (msaroufim) [reply README.md:77]: It's a term I've seen the compiler team use a bunch https://en.wikipedia.org/wiki/Code_generation_(compiler)

**Pull request merged**